app/src/main/java/com/example/movopfy/features/player/presentation/ui/PlayerScreen.kt

```diff
@@ -0,0 +1,39 @@
+package com.example.movopfy.features.player.presentation.ui
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import com.example.movopfy.features.player.presentation.viewmodel.PlayerViewModel
+import com.example.movopfy.uiComponents.components.ProgressBarLoading
+import org.koin.androidx.compose.koinViewModel
+
+@Composable
+fun PlayerScreen(
+    modifier: Modifier = Modifier,
+    id: Int,
+    episode: Int,
+    viewModel: PlayerViewModel = koinViewModel()
+) {
+    val uiState by viewModel.uiState.collectAsState()
+
+    LaunchedEffect(Unit) {
+        viewModel.getTitle(id = id)
```

Comment:
Are you sure, that you should fetch title from network? I think that you can pass it from DetailsScreen?

Reply:
yes